packages/instantsearch-ui-components/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,35 @@
+/** @jsx createElement */
+
+import { cx } from '../../lib/cx';
+
+import type { ComponentChildren, Renderer } from '../../types';
+
+export type AutocompleteProps = {
+  isOpen: boolean;
+  children?: ComponentChildren;
+  classNames?: Partial<AutocompleteClassNames>;
+};
+
+export type AutocompleteClassNames = {
+  /**
+   * Class names to apply to the root element
+   */
+  root: string | string[];
+};
+
+export function createAutocompleteComponent({ createElement }: Renderer) {
+  return function Autocomplete(userProps: AutocompleteProps) {
+    const { children, isOpen, classNames = {} } = userProps;
+
+    return (
+      <div
+        className={cx('ais-Autocomplete', classNames.root)}
+        role="combobox"
+        aria-expanded={isOpen}
+        aria-haspopup="listbox"
+      >
+        {children}
```

Comment:
in a later iteration (when we do the final dom) we likely want to have the search box be part of this too, and then all the intermediate components also be part of the autocomplete. For now this is a logical split.

Reply:
yes, there's a separate ticket for keyboard handling